packages/TagLabel/TagLabel.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div class="taglabel">
```

Comment:
we should avoid `div` for labels, they are supposed to be inlined.
just `<span><slot/></span>` should be enough

Reply:
Fixed